vivisect/parsers/__init__.py

```diff
@@ -97,3 +99,20 @@ def getBytesParser(fmt):
         import Elf
         return Elf.elfFromBytes
     return None
+
+
+ZIPKEY = 'filebytes'
+
+
+def compressBytes(byts):
+    with io.BytesIO() as fd:
+        with zipfile.ZipFile(fd, mode='w', compression=zipfile.ZIP_DEFLATED) as zipr:
```

Comment:
why zipfile vs gzip (which has a single default stream)?

Reply:
Mostly just trying to stay consistent with the couple other places in the code that use zipfile.

Reply:
which, if either, would be the most likely to be installed on any machine with python?

Reply:
on ubuntu:

```
$ dpkg -S /usr/lib/python3.8/zipfile.py
libpython3.8-minimal:amd64: /usr/lib/python3.8/zipfile.py
$ dpkg -S /usr/lib/python3.8/gzip.py
libpython3.8-stdlib:amd64: /usr/lib/python3.8/gzip.py
```

tbh, i don't know which is more likely to be installed, but i'm guessing it's libpython3.8-minimal?
but either way, they're both exceedingly likely to be installed on every system, so let's stick with what Vivisect already uses in other places.